src/main/java/TaskList.java

```diff
@@ -0,0 +1,46 @@
+public class TaskList {
+
+    private static final int MAX_TASKS = 100;
+
+    private static final Task[] tasks = new Task[MAX_TASKS];  // Array to store tasks
+    private static int tasksCount = 0;  // Counter to keep track of the number of tasks
+
+    public static void listTasks() {
+        Tommi.printLine();
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < tasksCount; i++) {
+            System.out.println((i + 1) + ". "  + tasks[i]);
+        }
+        Tommi.printLine();
+    }
+
+    public static void addTask(Task task) {
+        tasks[tasksCount] = task;
+        tasksCount++;
+        Tommi.printLine();
+        System.out.println("Sure. I've added the task: " + System.lineSeparator()
+                + task + System.lineSeparator()
+                + "There are now " + tasksCount + " tasks in the list.");
+        Tommi.printLine();
```

Comment:
The class is well structured, and the methods are concise and really easy to understand 😄. Would you want to switch a dynamic data structure later down the track (i.e. ArrayList), to give you some scalability? It might also be good to add boundary checks for adding tasks to ensure it doesn’t exceed MAX_TASKS. Overall, great foundation!